src/course_bot/general.clj

```diff
@@ -24,6 +24,7 @@
  {:en
   {:general
    {:who-am-i-3 "Name: %s; Group: %s; Telegram ID: %s"
+    :who-am-i-4 "Имя: %s; Группа: %s; Telegram ID: %s; Группа лаб.1: %s"
```

Comment:
Looks like a Russian text in English section.

Reply:
It's new commit. I am trying to improve whoami command. Now I have reset this commit, so you can merge my branch with fixing typos in russian words commit.